src/omv/imlib/imlib.h

```diff
@@ -341,20 +341,22 @@ typedef enum {
     PIXFORMAT_ID_BINARY     = 1,
     PIXFORMAT_ID_GRAY       = 2,
     PIXFORMAT_ID_RGB565     = 3,
-    PIXFORMAT_ID_YUV422     = 4,
-    PIXFORMAT_ID_BAYER      = 5,
+    PIXFORMAT_ID_BAYER      = 4,
+    PIXFORMAT_ID_YUV422     = 5,
     PIXFORMAT_ID_JPEG       = 6,
     /* Note: Update PIXFORMAT_IS_VALID when adding new formats */
 } pixformat_id_t;
 
 // Pixel sub-format IDs.
 typedef enum {
+    SUBFORMAT_ID_GRAY8      = 0,
+    SUBFORMAT_ID_GRAY16     = 1,
     SUBFORMAT_ID_BGGR       = 0, // !!! Note: Make sure bayer sub-formats don't  !!!
     SUBFORMAT_ID_GBRG       = 1, // !!! overflow the sensor.hw_flags.bayer field !!!
     SUBFORMAT_ID_GRBG       = 2,
     SUBFORMAT_ID_RGGB       = 3,
-    SUBFORMAT_ID_GRAY8      = 0,
-    SUBFORMAT_ID_GRAY16     = 1,
+    SUBFORMAT_ID_YUV422     = 0,
+    SUBFORMAT_ID_YVU422     = 1,
```

Comment:
You may want to add yuv flags to sensor_t hw_flags.

Reply:
Done.